src/main/java/projectLibraryPartTwo/service/AuthorService.java

```diff
@@ -0,0 +1,22 @@
+package projectLibraryPartTwo.service;
+
+import projectLibraryPartTwo.entity.Author;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class AuthorService extends Author{
```

Comment:
AuthorService should not extend Author, Author is entity, AuthorService is service logic class, there are different level in application. Also it is better create interface AuthorService, which includes method Author createAuthor(ResultSet resultSet) and then create implementation AuthorServiceImpl

Reply:
corrected